src/functionaltests/java/com/ericsson/ei/notifications/ttl/TestTTLSteps.java

```diff
@@ -0,0 +1,232 @@
+                    .insertDocument(missedNotificationDatabase,
+                            missedNotificationCollectionName,
+                            missedNotificationDocument.toString());
+            assertEquals(true, isInserted);
+
+            List<String> result = mongoDBHandler.getAllDocuments(missedNotificationDatabase,
+                    missedNotificationCollectionName);
+            assertEquals("Database " + missedNotificationDatabase + " is not empty.", false, result.isEmpty());
+        }
+
+        @Given("^Aggregated object is created in database with index \"([A-Za-z0-9_]+)\"$")
+        public void aggregated_object_is_created_in_database(
+            String indexName) throws IOException {
+            aggregatedObject = FileUtils.readFileToString(new File(AGGREGATED_OBJECT_FILE_PATH), "utf-8");
+            BasicDBObject aggregatedDocument = prepareDocumentWithIndex(aggregatedObject, indexName);
+
+            // setting 1 second TTL on index in db
+            mongoDBHandler.createTTLIndex(aggregationDataBaseName, aggregationCollectionName,
+                    indexName, 1);
+            Boolean isInserted = mongoDBHandler.insertDocument(aggregationDataBaseName,
+                    aggregationCollectionName, aggregatedDocument.toString());
+            assertEquals(true, isInserted);
+
+            List<String> result = mongoDBHandler.getAllDocuments(aggregationDataBaseName,
+                    aggregationCollectionName);
+            assertEquals("Database " + aggregationDataBaseName + " is not empty.", false, result.isEmpty());
+        }
+
+        @When("^I sleep for \"([^\"]*)\" seconds")
+        public void i_sleep_for_time(int sleepTime) throws Throwable {
+            LOGGER.debug("Sleeping for " + sleepTime + " at " + LocalTime.now());
+            // The background task deleting notifications from Mongo DB
+            // runs every 60 seconds
+            TimeUnit.SECONDS.sleep(sleepTime);
+        }
+
+        @Then("^\"([^\"]*)\" has been deleted from \"([A-Za-z0-9_]+)\" database$")
+        public void has_been_deleted_from_database(String collection, String database) {
+            LOGGER.debug("Checking " + collection + " in " + database);
+            List<String> result = mongoDBHandler.getAllDocuments(database, collection);
+            assertEquals("[]", result.toString());
+        }
+
+        // SCENARIO @TestNotificationRetries
+
+        @Given("^No missed notifications exists in database$")
+        public void no_missed_notifications_exists_in_database() {
+            mongoDBHandler.dropCollection(missedNotificationDatabase, missedNotificationCollectionName);
+        }
+
+        @Given("^Subscription is created$")
+        public void create_subscription() throws IOException, JSONException {
+            LOGGER.debug("Starting scenario @TestNotificationRetries.");
+            subscriptionObject = FileUtils.readFileToString(new File(SUBSCRIPTION_FILE_PATH), "utf-8");
```

Comment:
You should use "getJSONFromFile(filePath)" from the FunctionalTestBase instead of FileUtils.readFileToString(new File(SUBSCRIPTION_FILE_PATH), "utf-8");

This applies to all places in this code where you read a json-file.